packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/mapper/PomMapper.java

```diff
@@ -38,12 +38,6 @@ private Pom createAzurePom(Project project) {
 
         Set<String> addedDependencyPrefixes = new HashSet<>();
         List<String> dependencyIdentifiers = new ArrayList<>();
-        if (JavaSettings.getInstance().isStreamStyleSerialization()) {
-            addDependencyIdentifier(dependencyIdentifiers, addedDependencyPrefixes, Project.Dependency.AZURE_JSON,
-                false);
-            addDependencyIdentifier(dependencyIdentifiers, addedDependencyPrefixes, Project.Dependency.AZURE_XML,
-                false);
-        }
```

Comment:
azure-core pom includes them
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/pom.xml#L104-L113

Reply:
clientcore also contains json (I think no xml lib exists)
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/clientcore/core/pom.xml#L83-L87